packages/flipper-plugin-react-navigation/src/Logs.tsx

```diff
@@ -21,10 +21,11 @@ export function Logs({ active, logs, index, resetTo }: Props) {
       <DataList
         style={{ height: '100%' }}
         items={logs}
-        onRenderItem={({ id, action }, _, i) => (
+        selection={selectedID || ''}
```

Comment:
Empty string seems misleading. What's selected when passing an empty string? It should explicitly pass an ID (e.g. `selectedItem.id`)